main.go

```diff
@@ -51,6 +51,10 @@ type Config struct {
 	UseCpuShares            bool          `envconfig:"USE_CPU_SHARES" default:"false"`
 	Debug                   bool          `envconfig:"DEBUG" default:"false"`
 
+	// AWS Role options
+	AWSRole    string        `envconfig:"AWS_ROLE"`
+	AWSRoleTTL time.Duration `envconfig:"AWS_ROLE"`
```

Comment:
should this be `AWSRoleTTL time.Duration `envconfig:"AWS_ROLE_TTL"` `?

Reply:
Good catch!